test/utils/TestSession.js

```diff
@@ -89,7 +89,14 @@ class TestSession {
             knex.schema.dropTableIfExists('Dog_Movie'),
             knex.schema.dropTableIfExists('Person'),
             knex.schema.dropTableIfExists('Zombie'),
-            knex('knex_migrations').del()
+            knex.schema.hasTable('knex_migrations')
+                .then((exists) => {
```

Comment:
More promises sneaking in—we'll convert everything to async/await before long, but we need the library to settle down a bit before another refactor :)

Reply:
Yeah I completely forgot about .asCallback